src/rasenmaeher_api/web/api/healthcheck/views.py

```diff
@@ -13,10 +14,21 @@
 async def request_healthcheck() -> Dict[Any, Any]:
     """
     Basic health check. Success = 200. Checks the following things.
-    - Local db connection
+    - Person list from database
+    - Domain name from manifest
     """
-    Person.list()
-    return {"healthcheck": "success"}
+    # Do at least little bit something to check backend functionality
+    _ = Person.list()
+
+    # Get the DNS from manifest
+    my_dn: str = "Manifest not defined"
+    if settings.kraftwerk_manifest_bool:
+        if "dns" in settings.kraftwerk_manifest_dict:
+            my_dn = settings.kraftwerk_manifest_dict["dns"]
+        else:
+            my_dn = "DNS not defined in manifest"
+
+    return {"healthcheck": "success", "dns": my_dn}
```

Comment:
Should probably make a Pydantic schema for the response,

also call check_kraftwerk_manifest() from productapihelpers